src/clld_phylogeny_plugin/static/clld_phylogeny_plugin.js

```diff
@@ -85,14 +85,21 @@ CLLD_PHYLOGENY_PLUGIN.nodeStyler = function (labelSpec) {
             text = container.select("text");
             current = text.text();
             if (!current.endsWith(' ')) {
+                var values = [];
                 if (labelSpec.hasOwnProperty(current)) {
                     text.attr("fill", "red");
                     for (i = 0; i < labelSpec[current].length; i++) {
                         CLLD_PHYLOGENY_PLUGIN.marker(container, labelSpec[current][i], i);
+                        if (i == 0 && labelSpec[current][i].hasOwnProperty('tip_title')) {
+                          text.text(labelSpec[current][i]['tip_title']);
+                        }
+                        if (labelSpec[current][i].hasOwnProperty('tip_values')) {
+                          values.push(labelSpec[current][i]['tip_values']);
+                        }
                     }
                     text.attr("transform", null).attr ("x", function (d, i) { return labelSpec[current].length * 12;});
                 }
-                text.text(current + ' ');
+                text.text(text.text() + ' ' + values.join(" ◇ ") + ' ');
```

Comment:
The separator `◇` seems somewhat arbitrary. If so, it should be configurable (via `labelSpec`), I think.